### DISCUSSION TITLE
怎么修改数据库链接

### DISCUSSION THREAD

**Discussion by benbenc163** [lsky-org/lsky-pro#815] [问答]
请帮忙，事情是这样的，程序是部署在群辉docker里，群辉的IP修改了，数据库迁移，现在打开报504，IP改不回来，请问下怎么指向新的数据库IP，在哪个文件修改还是怎么修改，谢谢

**Comment by ghost** [comment]
https://github.com/lsky-org/lsky-pro/issues/643

修改.env文件
